problems/removeStringsEndingInBang.js

```diff
@@ -11,8 +11,11 @@
 * returns: ["", "a"]
 */
 
-function removeStringsEndingInBang() {
-
+function removeStringsEndingInBang(arr) {
+    return arr.filter((el) => {
+        el.lastIndexOf('!');
```

Comment:
This is a cool method you found, you just needed to compared it to see if the value returned is greater than negative 1 so `el.lastIndexOf('!') > -1`.You also forget to add the return statement at lube 16 to return the value back to filter function. If the fat arrow function was in one line then we don't need to add the return statement.

Reply:
Got it. Sparky thank you if you could share your code that i can understand why you will write them that would be great.. Thank you for all your feedbacks.

Reply:
Yea, actually now trying it myself this wouldn't work. oops.
Something simple could of been like this `return arr.filter((el) => el[el.length - 1] !== "!")`